src/ThirdParty/Github/GitHubConfigurator.php

```diff
@@ -14,111 +14,191 @@
 use Github\Client;
 use Github\Exception\TwoFactorAuthenticationRequiredException;
 use Gush\Adapter\DefaultConfigurator;
+use Gush\Helper\StyleHelper;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Console\Question\Question;
 
+/**
+ * @author Sebastiaan Stok <s.stok@rollerscapes.net>
+ */
 class GitHubConfigurator extends DefaultConfigurator
 {
     /**
+     * @var StyleHelper
+     */
+    private $styleHelper;
+
+    /**
+     * Constructor.
+     *
+     * @param StyleHelper $styleHelper QuestionHelper instance
+     * @param string      $label       Label of the Configurator (eg. GitHub or GitHub IssueTracker)
+     * @param string      $apiUrl      Default URL to API service (eg. 'https://api.github.com/')
+     * @param string      $repoUrl     Default URL to repository (eg. 'https://github.com')
+     */
+    public function __construct(StyleHelper $styleHelper, $label, $apiUrl, $repoUrl)
```

Comment:
concern is you are upgrading the github adapter but not the others. The other adapters will be more inconsistent if we merge this as it is. Can we update the other adapters as well?